services/libs/data-access-layer/src/members/affiliations.ts

```diff
@@ -10,59 +8,16 @@ export async function fetchMemberAffiliations(
   return qx.select(
     `
         SELECT
-          msa.id,
-          s.id as "segmentId",
-          s.slug as "segmentSlug",
-          s.name as "segmentName",
-          s."parentName" as "segmentParentName",
-          o.id as "organizationId",
-          o."displayName" as "organizationName",
-          o.logo as "organizationLogo",
-          msa."dateStart" as "dateStart",
-          msa."dateEnd" as "dateEnd"
-        FROM "memberSegmentAffiliations" msa
-               LEFT JOIN organizations o ON o.id = msa."organizationId"
-               JOIN segments s ON s.id = msa."segmentId"
-        WHERE msa."memberId" = $(memberId)
+          id,
+          "dateStart",
+          "dateEnd",
+          "organizationId",
+          "segmentId"
+        FROM "memberSegmentAffiliations"
+        WHERE "memberId" = $(memberId)
```

Comment:
`findMemberAffiliations()` in `services/libs/data-access-layer/src/member_segment_affiliations/index.ts` 🙄